templates/components/common/quick-search.html

```diff
@@ -5,7 +5,7 @@
                 {{lang "search.quick_search.input_label"}}
             </label>
             <input 
-                class="js-form-input js-search-quick"
+                class="js-form-input js-search-quick c-search-quick c-form__input"
```

Comment:
Seems like it should be `c-quick-search` (based on file name) + prevent mixing different components styles